### ISSUE TITLE
Server stability in France

### ISSUE THREAD

**Issue by ragibkl** [ragibkl/adblock-dns-server#63]
I'm having some stability issues with the server in France. I might need to resize it, or rebuild it from scratch.

This issue is to track any related notes to it.

**Comment by ragibkl** [comment]
@Tomatoide @TechComputerServiceDunkerque,

I'm going to have to setup a new server with more RAM. I think I can still keep the same ipv4 address, but the ipv6 will change unfortunately.

**Comment by ragibkl** [comment]
This is done now.
```shell
$ nslookup fr-dns1.bancuh.com 
Server:		127.0.0.53
Address:	127.0.0.53#53

Non-authoritative answer:
Name:	fr-dns1.bancuh.com
Address: 51.158.99.7
Name:	fr-dns1.bancuh.com
Address: 2001:bc8:600:1810::1
```

ipv4 is the same. ipv6 had to change. I'll keep this ticket open for a day or so.

**Comment by Tomatoide** [comment]
ipv6 looks the same as old one?

**Comment by TechComputerServiceDunkerque** [comment]
Hello, my adblock hosts stay the same?

**Comment by ragibkl** [comment]
> ipv6 looks the same as old one?

Oops, I ran that command too early. I've updated that, and it should reflect the new IP.

**Comment by TechComputerServiceDunkerque** [comment]
Thank you, sorry I didn't see that you answered. I also changed my text.

**Comment by ragibkl** [comment]
> Hello, my adblock hosts stay the same?
> 
> I saw thanks to Dns leak that you had put google back.
> 
> https://browserleaks.com/ip
> 
> It is also possible to change the Google dns on your server instead of this one? Thank you (a lot of french people don't
> trust Google).
> 
> anycast.censurfridns.dk
> 
> 91.239.100.100 
> 2001:67c:28a4::
> 
> Anycast from multiple locations.
> https://blog.uncensoreddns.org/

You are right. Thanks for catching that!

The servers that I setup has a default setting to use Google as forwarder. I have an override for that for the server in France. I forgot to add that back in when I recreated the server yesterday.

I'll update this now.

**Comment by TechComputerServiceDunkerque** [comment]
Thank you for your work

**Comment by ragibkl** [comment]
> Hello, my adblock hosts stay the same?
> 

Yes, all my servers use the same blocklist. Everytime we make an update to the list, each server should pull the latest version and restart.

Hopefully, with the added RAM, this France server will be able to auto-update as well, and not get stuck with the old list.

**Comment by TechComputerServiceDunkerque** [comment]
We will see ✌️

**Comment by TechComputerServiceDunkerque** [comment]
Bancuh doesn't work well in connection sharing, for example when I share my data with my smartphone, I think I got the wrong dns, I didn't give you the right one

ns0.fdn.fr : 80.67.169.12 ou 2001:910:800::12

ns1.fdn.fr : 80.67.169.40 ou 2001:910:800::40

**Comment by TechComputerServiceDunkerque** [comment]
I don't have a network at all with the French dns, on the other hand with the singapore dns, everything is fine.

**Comment by ragibkl** [comment]
> Bancuh doesn't work well in connection sharing, for example when I share my data with my smartphone, I think I got the wrong dns, I didn't give you the right one
> 
> ns0.fdn.fr : 80.67.169.12 ou 2001:910:800::12
> ns1.fdn.fr : 80.67.169.40 ou 2001:910:800::40
> Edit: 00h54 I'm still seeing cuts in the dns at the moment.

> I don't have a network at all with the French dns, on the other hand with the singapore dns, everything is fine.
> 
> is it possible to have a secondary dns?

I've applied the new ip forwarders to the France DNS. That's the only thing that is different from the singapore dns servers. Let's try this ip change for now, maybe it will recover the service.

If the problem is significantly more when you use connection sharing, I suspect the problem is related to rate limit. I'm trying to relax the rate-limits for now [here](https://github.com/ragibkl/adblock-dns-server/commit/6f5bb811b45b8a9bcecbe0322ad5ce21fdeef033). Full file [here](https://github.com/ragibkl/adblock-dns-server/blob/master/dns_base/templates/named.conf.options).
Do try out these changes, and feedback to me.

If possible, I don't want to deploy another server in France, cause I'm working on a tight budget. I'll think about it though.

**Comment by TechComputerServiceDunkerque** [comment]
Thank you, everything works now

**Comment by Tomatoide** [comment]
reminder to update bancuh website with updated dns 👍

**Comment by TechComputerServiceDunkerque** [comment]
Site to be unblocked

launches.appsflyer.com
fr.yahoo.com
7ba3f64df98de730df38846b54ecfbdf7f61f80f.cws.conviva.com
cws.conviva.com
app-measurement.com
measurement.com

Merci

**Comment by ragibkl** [comment]
> Site to be unblocked
> 
> launches.appsflyer.com
> fr.yahoo.com
> 7ba3f64df98de730df38846b54ecfbdf7f61f80f.cws.conviva.com
> cws.conviva.com
> app-measurement.com
> measurement.com
> 
> Merci

Hi @TechComputerServiceDunkerque ,

From your list, it seems that some of the sites are dead. There's no need to whitelist them I think.

Some of the sites are actual user tracking services. I don't think we should whitelist them. When I try to visit them, my browser adblock extension throws all kinds of security warnings.

`fr.yahoo.com`, is a legitimate site. I checked just now and it is not blocked.

**Comment by ragibkl** [comment]
Hi All. Thanks for your help in testing and ensuring the server in France work well.

Things seems to work fine, so I think we can close this issue. If you have other issues, please create a separate issue ticket.

Again, big thanks from me for your efforts!!!
